### DISCUSSION TITLE
Create record validation

### DISCUSSION THREAD

**Discussion by emageint** [filamentphp/filament#5220] [Help]
![chrome_screenshot_unitruc emage co uk](https://user-images.githubusercontent.com/19952221/206852847-f36bf51f-f433-4b97-ba29-bdb4d36b639d.png)

**Comment by Morry981** [comment]
https://filamentphp.com/docs/2.x/forms/validation
https://filamentphp.com/docs/2.x/forms/validation#custom-rules
